### ISSUE TITLE
Add support for aspect ratio

### ISSUE THREAD

**Issue by PedroSSM2000** [jaredh159/tailwind-react-native-classnames#142]
Tailwind comes with support for aspect ratio classes, like aspect-square video [custom] etc...

React Native also supports aspect ratio, why does tw`aspect-square` returns an empty object?

**Comment by jaredh159** [comment]
thanks for bringing this to my attention.  I think the `aspect-{ratio}` tailwind classes are new in V3 (or else I missed them previously).  the library already supports `aspect-ratio-{number}` utilities, but I just opened a PR to deprecate those, in favor of the v3 `aspect-{ratio}` utilities, and added support for `aspect-square` and `aspect-video`.  It doesn't look to me like RN Native supports `aspect-ratio: auto`, so I omitted that one.
